examples/MvcWeb/TheraLang/Controllers/ResourceController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using MvcWeb.TheraLang.Entities;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using MvcWeb.TheraLang.Services;
+
+namespace MvcWeb.TheraLang.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ResourceController : ControllerBase
+    {
+        public ResourceController(IResourceService _service)
+        {
+            this.service = _service;
+        }
+
+        private readonly IResourceService service;
+
+        [HttpPost]
+        public async Task<IActionResult> PostResource(Resource resource)
+        {
+            if(!ModelState.IsValid)
+            {
+                return BadRequest();
+            }
+
+            bool result = await service.AddResource(resource);
+            if(!result)
+            {
+                return BadRequest();
+            }
+
+            return Ok();
+        }
+
+        [HttpPut]
+        public async Task<IActionResult> PutResource(Resource resource, int updatedById)
```

Comment:
updated by you should take from the session, not the from UI :)
**Back-end should not believe what UI is saying - UI is the best liar** 😋 
Hmm, just noted that we don't have session implemented yet. Okay let it be for now.